src/redux/actions/projectActions.ts

```diff
@@ -70,7 +71,9 @@ export function saveProject(project: IProject)
             throw new AppError(ErrorCode.SecurityTokenNotFound, "Security Token Not Found");
         }
 
-        const savedProject = await projectService.save(project, projectToken);
+        const newProject = {...project, version: packageJson.version};
```

Comment:
Let's make sure we have a new test the validates the version is set when `saveProject` is called.

Reply:
Added test for both saveProject and saveAssetMetadata